### ISSUE TITLE
Remove test folder sdist generated package

### ISSUE THREAD

**Issue by shinneider** [shinneider/django-admin-search#19]
Package generated by `python setup.py sdist` contains a test folder inside, but is not necessary for work, i want to remove this.

**Comment by shinneider** [comment]
Possible solution: replace all `MANIFEST.in` by this:
```
include LICENSE
include README.md

global-exclude __pycache__
global-exclude *.py[co]
global-exclude tests *

recursive-include django_admin_search/templates *
recursive-include django_admin_search/static *
recursive-include django_admin_search/locale *
```

**Comment by shinneider** [comment]
Resolved on df5d50767b562171f9285bee01d2d121e84d089e
